pygsti/protocols/gst.py

```diff
@@ -2195,7 +2370,7 @@ def _evaluate_constraints(wv):
         # Note: active_constraints_list is typically stored in parameters['unmodeled_error active constraints']
         # of the relevant Estimate object.
         primOp_labels = _collections.defaultdict(list)
-        for lbl, i in budget.primOpLookup.items(): primOp_labels[i].append(str(lbl))
+        for lbl, i in budget.primitive_op_vecindex.items(): primOp_labels[i].append(str(lbl))
```

Comment:
This line causes tests to fail. Should it just be `primitive_op_index`, to match the new member variable in the base class?
```
======================================================================
1711) ERROR: test_add_badfit_estimates (test.unit.protocols.test_gst.GSTUtilTester)
----------------------------------------------------------------------
   Traceback (most recent call last):
    protocols\test_gst.py line 71 in test_add_badfit_estimates
      gst._add_badfit_estimates(res, 'test-estimate', badfit_opts, opt)
    C:\hostedtoolcache\windows\Python\3.7.9\x64\lib\site-packages\pygsti\protocols\gst.py line 1874 in _add_badfit_estimates
      budget_dict = _compute_wildcard_budget(objfn_cache, mdc_objfn, parameters, badfit_options, printer - 1)
    C:\hostedtoolcache\windows\Python\3.7.9\x64\lib\site-packages\pygsti\protocols\gst.py line 2373 in _compute_wildcard_budget
      for lbl, i in budget.primitive_op_vecindex.items(): primOp_labels[i].append(str(lbl))
   AttributeError: 'NoneType' object has no attribute 'items'
   ```

Reply:
Checked this locally and the misnaming appears to be the case. We'll see if the latest set of tests pass.